app/javascript/components/EditForm.js

```diff
@@ -38,7 +38,7 @@ class EditRebateForm extends React.Component {
   onSubmit (values) {
     return values.newIncomeField
       ? this.addNewIncomeValue(values)
-      : fetch(`${appUrl}admin/rebate_forms/${this.props.rebateForm.id}`, {
+      : fetch(`${appUrl}/admin/rebate_forms/${this.props.rebateForm.id}`, {
```

Comment:
hmm.. wouldn't this work fine if you left out the ${appUrl} entirely so it was a relative path on the current site?

Reply:
Yes you can omit it, but it should be absolute instead. Otherwise the url gets appended.